src/openlifu/db/database.py

```diff
@@ -274,6 +276,50 @@ def write_volume(self, subject_id, volume_id, volume_name, volume_data_filepath,
 
         self.logger.info(f"Added volume with ID {volume_id} for subject {subject_id} to the database.")
 
+    def write_photoscan(self, session: Session, photoscan_id, photoscan_name, model_data_filepath, texture_data_filepath, mtl_data_filepath: Optional[Path] = None, on_conflict=OnConflictOpts.ERROR):
+        if not Path(model_data_filepath).exists():
+            raise ValueError(f'Model data filepath does not exist: {model_data_filepath}')
+        if not Path(texture_data_filepath).exists():
+            raise ValueError(f'Texture data filepath does not exist: {texture_data_filepath}')
```

Comment:
You [can test](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions) that these are working in your unit tests by doing
```py
with pytest.raises(ValueError, match="data filepath does not"):
  ...
```
If the code in the ... does not raise a ValueError with the `match` text in its message, then the test fails.

Unrelated: also a `FileNotFoundError` may be more suitable than `ValueError` here